test/cypress/integration/getting_started_spec.ts

```diff
@@ -0,0 +1,330 @@
+import { escapeRegExp, find } from 'lodash';
+import pluginsCatalog from '../fixtures/plugins/catalog.json';
+
+const awsSecrets = ['aws_access_key_id', 'aws_secret_access_key'];
+const awsPlugins = ['cloudify-utilities-plugin', 'cloudify-kubernetes-plugin', 'cloudify-aws-plugin'];
+const awsBlueprints = ['AWS-Basics-VM-Setup', 'AWS-VM-Setup-using-CloudFormation', 'Kubernetes-AWS-EKS'];
+
+const gcpSecrets = [
+    'gcp_client_x509_cert_url',
+    'gcp_client_email',
+    'gcp_client_id',
+    'gcp_project_id',
+    'gcp_private_key_id',
+    'gcp_private_key',
+    'gcp_zone'
+];
+
+const goToBackStep = () => cy.contains('button', 'Back').click();
+const goToNextStep = () => cy.contains('button', 'Next').click();
+const goToFinishStep = () => cy.contains('button', 'Finish').click();
+const closeModal = () => cy.contains('button', 'Close').click();
+
+function verifyInstallationSucceed(blueprints: string[]) {
+    cy.contains('.progress .progress', '100%', { timeout: blueprints.length * 2 * 60 * 1000 });
+    cy.contains('.progress .label', 'Installation done!');
+    cy.get('.ui.red.message').should('not.exist'); // there shouldn't be visible error messages
```

Comment:
Not sure if we need the comment :smile: The assertion is self-explanatory to me

```suggestion
    cy.get('.ui.red.message').should('not.exist');
```